src/openrct2/scripting/bindings/entity/ScGuest.cpp

```diff
@@ -472,6 +475,32 @@ namespace OpenRCT2::Scripting
         }
     }
 
+    std::shared_ptr<ScRide> ScGuest::rideHeadedTo_get() const
+    {
+        ThrowIfGameStateNotMutable();
```

Comment:
`ThrowIfGameStateNotMutable()` only needs to be called when setting data, not getting.